addons/web_tip/static/src/js/tip.js

```diff
@@ -250,7 +242,13 @@
                         def.resolve();
                     }
                 });
-
+                self.$element.on('mousedown', function($ev) {
+                    if (tip.mode == 'kanban')
+                    {
+                        self.end_tip(tip);
+                        def.resolve();
+                    }
+                });
```

Comment:
The proper way to do this is in the tip definition. Use `end_selector` and `end_event`